tests/helixpublish.proj

```diff
@@ -76,15 +76,15 @@
 
     <ItemGroup>      
       <HelixCorrelationPayloadFile Include="$(ArchivesRoot)Core_Root*/*.zip">
-        <RelativeBlobPath>$(Platform)$(ConfigurationGroup)/%(Identity)</RelativeBlobPath>
+        <RelativeBlobPath>$(Platform)$(BuildType)/%(Identity)</RelativeBlobPath>
       </HelixCorrelationPayloadFile>
       <HelixWorkItem>
         <Command Condition="'$(TargetsWindows)' == 'true'">$(HelixPythonPath) $(RunnerScript) --script %HELIX_WORKITEM_PAYLOAD%\runtests.cmd</Command>
         <Command Condition="'$(TargetsWindows)' != 'true'">chmod +x $HELIX_WORKITEM_PAYLOAD/runtests.sh &amp;&amp; $(HelixPythonPath) $(RunnerScript) --script $HELIX_WORKITEM_PAYLOAD/runtests.sh</Command>
         <PayloadFile>%(Identity)</PayloadFile>
         <WorkItemId>%(Filename)</WorkItemId>
         <TimeoutInSeconds>$(TimeoutInSeconds)</TimeoutInSeconds>
-        <RelativeBlobPath>$(Platform)-$(ConfigurationGroup)/Tests/$([System.String]::Copy('%(RecursiveDir)').Replace('\', '/'))%(Filename)%(Extension)</RelativeBlobPath>
+        <RelativeBlobPath>$(Platform)-$(BuildType)-$(Rid)/Tests/$([System.String]::Copy('%(RecursiveDir)').Replace('\', '/'))%(Filename)%(Extension)</RelativeBlobPath>
```

Comment:
What's the motivation for this change?

Reply:
ConfigurationGroup is unset now, leading to uploading all tests for all platforms to paths like x64- and arm- folders.  This was causing massive file stomping, leading to almost all tests failing (it was a random race condition thing)

Reply:
It is a little more confusing than this. As just having the configuration after the arch is also not sufficient enough to guarantee independent resources.

Reply:
@jashook was the variable not also unset?

Reply:
Looks as if the variable has been under for a while. The cause instead seems to be linked to a change in pb_container name, which used to have the os.